src/coord/catalog.rs

```diff
@@ -568,20 +535,13 @@ impl Catalog {
     ) -> Result<&mut Schema, Error> {
         // Keep in sync with `get_schemas`.
         match database_spec {
+            DatabaseSpecifier::Ambient if schema_name == "mz_temp" => {
```

Comment:
Should we pull `mz_temp` into a constant somewhere?

Reply:
Good idea! Done.